### ISSUE TITLE
Testing on Azure needed

### ISSUE THREAD

**Issue by Ulexus** [CyCoreSystems/netdiscover#1]
The Azure implementation has been added purely on the documentation.  Live testing needs to be done to validate it.

**Comment by juansalvatella** [comment]
I've recently done a test in Azure and it's not working properly. I'll have a look and submit a pull request if I find out what's wrong.

**Comment by Ulexus** [comment]
I have used this on Azure before; I recall the results being weird, but I do not recall exactly how.

**Comment by juansalvatella** [comment]
Could it be that the Azure Kubernetes Service doesn't provision a public IP to the nodes in the cluster? I'd say that's why  netdiscover is not picking up the IP... there is none.

**Comment by Ulexus** [comment]
Yes, that is certainly true of the default node setup.  However, you _can_ set them to get a public IP.  I seem to recall that even in that case, something was weird.
